src/3k-mapper.xml

```diff
@@ -1154,6 +1169,170 @@ end</script>
                     <packageName></packageName>
                     <regex>^tmap addalias &quot;(.+)&quot; (.+)$</regex>
                 </Alias>
+                <Alias isActive="yes" isFolder="no">
+                    <name>Rename an area</name>
+                    <script>--renames current area
+local Name = matches[2]
+local cArea = getRoomArea(tmap.lastId)
+setAreaName( cArea, Name )</script>
+                    <command></command>
+                    <packageName></packageName>
+                    <regex>^tmap renamearea (.+)$</regex>
+                </Alias>
+                <Alias isActive="yes" isFolder="no">
+                    <name>Connect all stub in area</name>
+                    <script>local aid = getRoomArea(tmap.lastId)
+local rooms = getAreaRooms(aid)
+for i,v in pairs(rooms) do
+	local stubs = getExitStubs(v)
```

Comment:
`getExitStubs( roomId )` starts its listing with a `0` - but there is now `getExitStubs1( roomId )` which does the correct thing for a lua table and starts at 1 for the first entry.